packages/kit/rollup.config.js

```diff
@@ -48,7 +48,8 @@ export default [
 			cli: 'src/cli.js',
 			ssr: 'src/runtime/server/index.js',
 			node: 'src/core/node/index.js',
-			'install-fetch': 'src/install-fetch.js'
+			'install-fetch': 'src/install-fetch.js',
+			utils: 'src/core/utils.js'
```

Comment:
I wonder if we should make this `adapter-utils` with just the one new method rather than publishing all of our utils

Reply:
Yeah it makes more sense I just didn't know which name to use until now